### DISCUSSION TITLE
Inquiring about video upload capability when listing products

### DISCUSSION THREAD

**Discussion by bobyagnpf** [amzn/selling-partner-api-models#3669] [General] [answered]
Could you please clarify if there is an option to include videos as part of the product listing through the Selling Partner API? If so, could you provide guidance or documentation on how to use the video upload feature?

**Comment by rugved1991** [comment, marked as answer]
Hi @bobyagnpf,

Currently we do not support videos on the Listings through the API. You will have to continue using the Seller Central UI for now. I have marked this as a feature request. 
You can refer to these documentation for more information on how to upload videos on seller central:
https://sellercentral.amazon.com/help/hub/reference/GWJRQF3C34G4PRA2
https://sellercentral.amazon.com/help/hub/reference/GSL5BGQC2EYYDQ29
https://www.youtube.com/watch?v=A0_iFz7i56g

Best,
Rugved
Solutions Architect, SP API

**Reply by 75alman** [reply]
I'm following the progress of the video upload feature ( Issue amzn/selling-partner-api-docs#3305 ) with great interest. Could you please provide an update on the current status?

**Reply by chagasjoao** [reply]
I'm looking for this implementation too!

**Reply by Azad2314** [reply]
Hi @rugved1991,

Thank you for the information. I wanted to check if there have been any updates on this feature. Is video support for Listings available now via the API, or is it still not available for use?


Looking forward to your response.

**Reply by ritikacd** [reply]
Hi @Azad2314 

This feature is still not supported by SP-API yet. It is still under the review. Once we receive additional information, we will post an update. 

Thanks
Ritika
Selling Partner API Services

**Reply by xuqiang97** [reply]
Any updates ?

**Comment by gmiretti** [comment]
Also it would be interesting that you offer a report type in Report API to get the video traffic metrics

**Comment by xuqiang97** [comment]
Any updates ?

**Reply by ritikacd** [reply]
Hi @xuqiang97 The feature is still not supported through SP-API. Once it's available, we will notify users through documentation [release notes](https://developer-docs.amazon.com/sp-api/docs/sp-api-release-notes). 

Thank you,
Ritika

**Comment by xuqiang97** [comment]
Any updates ?
